lib/ide/ast/impl/Cargo.toml

```diff
@@ -10,6 +10,8 @@ crate-type = ["cdylib", "rlib"]
 [dependencies]
 derive_more  = { version = "0.15.0"                             }
 failure      = { version = "0.1.5"                              }
+lazy_static =  { version = "1.4.0"                              }
+regex        = { version = "1"                                  }
```

Comment:
I believe this increases our package size drastically. I was trying to include it several times already and always I was reverting it. Why do you need it for?

Reply:
If we can trust `script/assert_max_wasm_size.sh`, then there is no visible size increase.

Reply:
I checked it, the gui.wasm.gz from this branch is 7 bytes greater than from master.

It may be, that regex library is already pulled as dependency somewhere (perhaps by another thirdparty)